kalite/topic_tools/__init__.py

```diff
@@ -67,6 +78,20 @@ def get_topic_tree(force=False, annotate=False, channel=None, language=None, par
         TOPICS = {}
     if TOPICS.get(channel) is None:
         TOPICS[channel] = {}
+
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
```

Comment:
This PR was about fixing stuff that was already broken in some code that I didn't write, I don't feel much like cleaning up the code.. it's a bit like combing the desert, if you know what I mean... I can copy paste your above suggestion, but I also think we should move on :) thanks for the suggestion.